src/lib/flatten/rollup.ts

```diff
@@ -49,13 +52,29 @@ export async function rollupBundleFile(opts: RollupOptions): Promise<void> {
   });
 
   // Output the bundle to disk
-  await bundle.write({
-    name: `${opts.moduleName}`,
-    file: opts.dest,
+  const result = await bundle.generate({
+    name: opts.moduleName,
     format: opts.format,
     amd: opts.amd,
     banner: '',
     globals: moduleId => umdModuleIdStrategy(moduleId, opts.umdModuleIds || {}),
     sourcemap: true
```

Comment:
For rollup, we'd need to change the output options:

```diff
diff --git a/src/lib/flatten/rollup.ts b/src/lib/flatten/rollup.ts
index 67ee77b..1dc459f 100644
--- a/src/lib/flatten/rollup.ts
+++ b/src/lib/flatten/rollup.ts@@ -52,13 +52,16 @@ export async function rollupBundleFile(opts: RollupOptions): Promise<void[]> {
   });

   // Output the bundle to disk
+  const sourcemapFile = `${opts.dest}.map`;
   const result = await bundle.generate({
+    file: opts.dest,
     name: opts.moduleName,
     format: opts.format,
     amd: opts.amd,
     banner: '',
     globals: moduleId => umdModuleIdStrategy(moduleId, opts.umdModuleIds || {}),-    sourcemap: true
+    sourcemap: 'inline',
+    sourcemapFile
   });

   // relocate sourcemaps
@@ -76,5 +79,5 @@ export async function rollupBundleFile(opts: RollupOptions): Promise<void[]> {     }
   });

-  return Promise.all([outputJson(`${opts.dest}.map`, result.map), outputFile(opts.dest, result.code)]);
+  return Promise.all([outputJson(sourcemapFile, result.map), outputFile(opts.dest, result.code)]); }
```


Still then, we have to append the final line ourselves as [rollup does it in their `build` executable command](https://github.com/rollup/rollup/blob/master/bin/src/run/build.ts#L59-L63)

Reply:
See https://github.com/rollup/rollup/issues/121

Reply:
Yeah, realised that. Thanks for your help. I’ll do these within the hour

Reply:
I pushed an update

Reply:
bdw, kudos for realising that `sourceMappingURL` was not being added